dev/breeze/src/airflow_breeze/global_constants.py

```diff
@@ -73,10 +77,18 @@
 #   - https://endoflife.date/azure-kubernetes-service
 #   - https://endoflife.date/google-kubernetes-engine
 ALLOWED_KUBERNETES_VERSIONS = ["v1.23.13", "v1.24.7", "v1.25.3", "v1.26.0"]
+
 ALLOWED_EXECUTORS = ["KubernetesExecutor", "CeleryExecutor", "LocalExecutor", "CeleryKubernetesExecutor"]
```

Comment:
@potiuk Can you please confirm if the list for ALLOWED_EXECUTORS is good enough for the `--executor` flag? I think it should also have `SequentialExecutor` which was directly mapped to `SQLite` in  `scripts/ci/docker-compose/backend-sqlite.yml`. But I'm not sure if it will have any side effects or not. WDYT?

Reply:
I think we should simply automatically force the executor to be sequential for sqlite and it should not be in the list. It's not useful for any other backend.

Reply:
Sure, I'll change the code accordingly.